webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -749,6 +749,11 @@ Kommunicate.richMsgEventHandler = {
         console.log('passenger detail submitted');
     },
     processQuickReplies: function (e) {
+        window.KM_WidgetEvents.onRichMessageButtonClick('Rich button clicked');
```

Comment:
check if the google analytics function can be made from KM_WidgetEvents itself, so that all places we are making only 1 common function call